nss-day-cohort-39/hello-world-the-globtrotters
# Automated lists on Thailand page & CSS styling

### Changes to Thailand page:
- Automated lists of cities, landmarks, and heads of states
- Added footer with student info
- CSS Styling: changed colors, added border, styled header

### Testing
- Make sure all styling changes are on Thailand page
- Check for footer on Thailand page
- Make sure all links work correctly
